### ISSUE TITLE
[UIVeri5]: Selection of default reporter JUNIT as additional reporter not needed

### ISSUE THREAD

**Issue by georgbischoff** [SAP/generator-easy-ui5#62]
Easy-Ui5 version: generator-easy-ui5@2.4.4

OS/version: MacOS BigSur

Browser/version (+device/version): Google Chrome

Steps to reproduce the problem:
1. Run the generator for UIVeri5

What is the expected result?
As JUnit is already one of the default reporters I would not expect to add it in the uiveri5 generator as additional reporter.
Instead I would expect an option: none

What happens instead?
You are asked to add JUNIT or SauceLabs.

Any other information? (attach screenshot if possible)

**Comment by tsaleksandrova** [comment]
Hi Georg,
afaik, only the console and screenshot reporters are enabled by default (for the integration profile) and are not in the list of additional reporters (also mentioned here https://github.com/SAP/ui5-uiveri5/blob/master/docs/config/reporters.md)

**Comment by georgbischoff** [comment]
Hi Tsvetelina,
thanks, understood but in this case it is not generating any additional properties. If I select JUnit my conf.js file has only profile and baseUrl but nothing about JUnit:
```
exports.config = {
  profile: "integration",
  baseUrl: "http://localhost:8080"
};
```
I would still expect the option "none" if s.o. is fine with the default screenshot reporter and does not need anything else.

Regards,
Georg

**Comment by IObert** [comment]
The fix is included in version 2.4.6 which is why I close the issue now.
Please reopen if something is not working as expected.
